pom.xml

```diff
@@ -269,6 +269,11 @@
             <version>1.14</version>
             <type>pom</type>
         </dependency>
+        <dependency>
```

Comment:
add to  [Deps](https://github.com/netgrif/application-engine/pull/101#:~:text=Implements%20NAE%2D1693-,Dependencies,-No%20new%20dependencies)